src/databricks/labs/remorph/install.py

```diff
@@ -298,6 +305,12 @@ def _save_config(self, config: MorphConfig | ReconcileConfig):
         if self._prompts.confirm(f"Open config file {ws_file_url} in the browser?"):
             webbrowser.open(ws_file_url)
 
+    def _apply_upgrades(self):
+        try:
+            self._upgrades.apply(self._ws)
+        except (InvalidParameterValue, NotFound) as err:
+            logger.warning(f"Installed version is too old: {err}")
```

Comment:
this warning is not correct - there may be other reasons.

Reply:
Updated